example/collector-k8-manifest.yml

```diff
@@ -11,9 +11,8 @@ spec:
   config: |
     receivers:
       kubearmor_receiver:
-        endpoint: ${env:KUBEARMOR_SERVICE?default=kubearmor:32767}
-        logfilter: ${env:LOGFILTER?default=all}
```

Comment:
Hmm.... So I did this so that we could have a way to override the values we set in the config file present in our image. [Ref](https://github.com/kubearmor/otel-adapter/blob/9f702ef1f52dd6e161061908b2bc884f281bb473/Dockerfile#L24).
As for some reason the environment variable value read [here](https://github.com/kubearmor/otel-adapter/blob/9f702ef1f52dd6e161061908b2bc884f281bb473/stanza_input_operator/input_operator.go#L36) was overridden at some later stage.

Thus, if someone used docker, they could now just override as intended with:
`docker run -e KUBEARMOR_SERVICE=kubearmor:1234 kubearmor/otel-receiver`

But yes, doing the same for K8s doesn't seem right since the entire configuration is recreated from the spec anyways. So looking good. :+1:

Reply:
Just one more change.
Can you also remove the defaults from here? - https://github.com/kubearmor/otel-adapter/blob/9f702ef1f52dd6e161061908b2bc884f281bb473/example/config.yml#L3-L4
So we'll just have `${env:KUBEARMOR_SERVICE}` and so on...

Apparently, there is no way to set defaults for env variables rn. It was bad research from my side : (

Reply:
Thanks for the update and the review.